common/testing/taskpoller/taskpoller.go

```diff
@@ -95,7 +97,7 @@ var (
 )
 
 func New(
-	t *testing.T,
+	t Helper,
```

Comment:
This is to make sure that `TaskPoller` doesn't need to be reinitialized for subtests.